### ISSUE TITLE
Still doesn't seem to work on 2.0.2

### ISSUE THREAD

**Issue by mba08mc** [minif/iphoneos2-app-fixer#3]
I'm attempting to install a .ipa on iPhoneOS 2.0.2 yet I still receive the same error. I updated com.apple.mobile.installation.composite_trust.plist /var/mobile/Library/Caches and restarted the device. The sh script didn't seem to modify the info.plist within the .ipa when I extracted it so I manuall added in the child entry myself. After repackaging the payload it still seems to give the error that the application can't be opened. Keep in mind that I've also installed the patched MobileInstallation v2.0.2 file. The application in question is Facebook 1.0. Here is the manually patched .ipa so maybe you could look into it. Also, what device and version do you use to test this, I may resort to updating to 2.2.1 if nothing else works.
[fb.ipa.zip](https://github.com/user-attachments/files/16120836/fb.ipa.zip)

**Comment by mba08mc** [comment]
Furthermore, I can verify that the app itself isn't corrupt since it runs fine on my iOS 4 device

**Comment by minif** [comment]
Thanks for letting me know, I just tested it on my iPod touch 2G on iPhone OS 2.2.1 and it works perfectly fine, so I think it might be an issue with older versions. For now I'll put a disclaimer.

**Comment by minif** [comment]
Eventually if I can test with iPhoneOS 2.0.2 (I will need another device) I'll see what is different, and if I can find a solution I'll update it here.

**Comment by mba08mc** [comment]
Yeah, I'm guessing at some point they changed the way applications were signed. I'll try 2.1, 2.2 and 2.2.1 and maybe an earlier version and see if those work. I'm hopeful it's to do with the firmware and not the fact I'm on a iTouch 1G

**Comment by mba08mc** [comment]
Updayed my iPod to 2.1, it seems to work perfectly fine now. I'm guessing this is a problem with devices running earlier than 2.1 but I'll try testing again in the future.
